modules/docs/src/main/mdoc/docs/14-Managing-Connections.md

```diff
@@ -121,7 +121,7 @@ object HikariApp extends IOApp {
 And running this program gives us the desired result.
 
 ```scala mdoc
-HikariApp.main(Array())
+// HikariApp.main(Array()) // https://github.com/typelevel/cats-effect/issues/1560
```

Comment:
Is this now resolved?

Reply:
I just did a quick test and unfortunately after uncommenting this line rendering the mdoc does not terminate.

from stack dump:
```
"run-main-0" #486 prio=5 os_prio=0 cpu=18472,29ms elapsed=1078,35s tid=0x00007f083403e800 nid=0x48714 waiting on condition  [0x00007f07c6ffa000]
   java.lang.Thread.State: TIMED_WAITING (parking)
        at jdk.internal.misc.Unsafe.park(java.base@11.0.8/Native Method)
        - parking to wait for  <0x00000004246a7340> (a java.util.concurrent.CountDownLatch$Sync)
        at java.util.concurrent.locks.LockSupport.parkNanos(java.base@11.0.8/LockSupport.java:234)
        at java.util.concurrent.locks.AbstractQueuedSynchronizer.doAcquireSharedNanos(java.base@11.0.8/AbstractQueuedSynchronizer.java:1079)
        at java.util.concurrent.locks.AbstractQueuedSynchronizer.tryAcquireSharedNanos(java.base@11.0.8/AbstractQueuedSynchronizer.java:1369)
        at java.util.concurrent.CountDownLatch.await(java.base@11.0.8/CountDownLatch.java:278)
        at cats.effect.IOPlatform.unsafeRunTimed(IOPlatform.scala:40)
        at cats.effect.IOPlatform.unsafeRunSync(IOPlatform.scala:27)
        at repl.MdocSession$App.<init>(15-Extensions-PostgreSQL.md:78)
        at repl.MdocSession$.app(15-Extensions-PostgreSQL.md:3)
        at mdoc.internal.document.DocumentBuilder$$doc$.$anonfun$build$2(DocumentBuilder.scala:89)
```
No other thread appears to be holding `0x00000004282929d8`

Reply:
Wait, I just noticed that the file name in the stack trace is wrong? An also line numbers! lines 3 and 78 in `15-Extensions-PostgreSQL.md` do not contain runable code and I'm positive that it's the line 124 in `14-Managing-Connections.md` that makes the rendering hang or not.